wheel/generate_type_stubs.py

```diff
@@ -305,6 +305,27 @@ def confirm_not_included_already_hashed(
     proof: bytes,
 ) -> bool: ...
 
+def validate_clvm_and_signature(
+    new_spend: SpendBundle,
+    max_cost: int,
+    constants: ConsensusConstants,
+    peak_height: int,
+    cache: Option[BLSCache],
```

Comment:
I don't think you should pass in the cache here. If we want to make that change, I think it should be considered independently of porting this to rust